app/frontend/websocket/configure.js

```diff
@@ -51,48 +52,27 @@ const configureWebSocket = async (server, funcLogger) => {
 
       const commandLogger = logger.child({ payload });
 
-      switch (payload.command) {
-        case 'latest':
-          await handleLatest(commandLogger, ws, payload);
-          break;
-        case 'setting-update':
-          await handleSettingUpdate(commandLogger, ws, payload);
-          break;
-        case 'symbol-update-last-buy-price':
-          await handleSymbolUpdateLastBuyPrice(commandLogger, ws, payload);
-          break;
-        case 'symbol-delete':
-          await handleSymbolDelete(commandLogger, ws, payload);
-          break;
-        case 'symbol-setting-update':
-          await handleSymbolSettingUpdate(commandLogger, ws, payload);
-          break;
-        case 'symbol-setting-delete':
-          await handleSymbolSettingDelete(commandLogger, ws, payload);
-          break;
-        case 'symbol-grid-trade-delete':
-          await handleSymbolGridTradeDelete(commandLogger, ws, payload);
-          break;
-        case 'symbol-enable-action':
-          await handleSymbolEnableAction(commandLogger, ws, payload);
-          break;
-        case 'manual-trade':
-          await handleManualTrade(commandLogger, ws, payload);
-          break;
-        case 'manual-trade-all-symbols':
-          await handleManualTradeAllSymbols(commandLogger, ws, payload);
-          break;
-        case 'cancel-order':
-          await handleCancelOrder(commandLogger, ws, payload);
-          break;
-        case 'dust-transfer-get':
-          await handleDustTransferGet(commandLogger, ws, payload);
-          break;
-        case 'dust-transfer-execute':
-          await handleDustTransferExecute(commandLogger, ws, payload);
-          break;
-        default:
-          handleWarning(logger, ws, 'Command is not recognised.');
+      const commandMaps = {
+        latest: handleLatest,
+        'setting-update': handleSettingUpdate,
+        'symbol-update-last-buy-price': handleSymbolUpdateLastBuyPrice,
+        'symbol-delete': handleSymbolDelete,
+        'symbol-setting-update': handleSymbolSettingUpdate,
+        'symbol-setting-delete': handleSymbolSettingDelete,
+        'symbol-grid-trade-delete': handleSymbolGridTradeDelete,
+        'symbol-enable-action': handleSymbolEnableAction,
+        'symbol-trigger-buy': handleSymbolTriggerBuy,
+        'manual-trade': handleManualTrade,
+        'manual-trade-all-symbols': handleManualTradeAllSymbols,
+        'cancel-order': handleCancelOrder,
+        'dust-transfer-get': handleDustTransferGet,
+        'dust-transfer-execute': handleDustTransferExecute
+      };
+
+      if (commandMaps[payload.command]) {
+        await commandMaps[payload.command](commandLogger, ws, payload);
+      } else {
+        handleWarning(logger, ws, 'Command is not recognised.');
```

Comment:
In line 49 there is another `handleWarning` call which is wrapped in `ws.send` and it should be edited because `handleWarning` already has `ws.send`

Reply:
Oh damn. Good find. Thanks!!